src/HeuristicCompletion-Benchmarks-Tests/CoStaticBenchmarksTest.class.st

```diff
@@ -0,0 +1,329 @@
+Class {
+	#name : 'CoStaticBenchmarksTest',
+	#superclass : 'TestCase',
+	#instVars : [
+		'benchmarks'
+	],
+	#category : 'HeuristicCompletion-Benchmarks-Tests',
+	#package : 'HeuristicCompletion-Benchmarks-Tests'
+}
+
+{ #category : 'running' }
+CoStaticBenchmarksTest >> setUp [ 
+
+	super setUp.
+	benchmarks := CoStaticBenchmarks new.
+	benchmarks builder: CoGlobalSorterResultSetBuilder new.
+	benchmarks initialize.
```

Comment:
This is strange to call initialize explicitly because it was called by new already and if builder: as side effects the initialize call can break them.